root/etc/s6-overlay/s6-rc.d/init-mod-code-server-php-cli-install/run

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/with-contenv bash
+
+echo "**** Setting up PHP configuration files ****"
+if [ ! -d /config/.php ]; then
+    cp -rf /etc/php /config/.php
+    chown -R abc:abc /config/.php
```

Comment:
please use our `lsiown` alias instead of `chown`, which works better with remote or fuse mounts

Reply:
Is fixed